charts/prefect-orion/values.yaml

```diff
@@ -35,6 +35,7 @@ orion:
   # -- array with environment variables to add to orion nodes
   env: []
   ## env:
+  ##   -- API url used by services such as the agent; can be in-cluster API url
```

Comment:
Since this is in the Orion chart (not the agent chart) does this make sense? Is the UI API URL referenced anywhere in this chart? This is where it's relevant.